packages/test/test-service-load/src/leaderElection.ts

```diff
@@ -32,6 +35,7 @@ export class LeaderElection {
         });
 
         this.dataStoreRuntime.on("connected", () => {
+            clearInterval(interval);
```

Comment:
When do we need this if you aread already checking before?

Reply:
I don't know that we guarantee we won't emit "connected" twice in a row, in which case this code as is would abandon its interval which would keep sending signals even when disconnected/disposed.